src/workers/job-processes.js

```diff
@@ -351,11 +357,23 @@ const syncProcessMap = {
 export async function dispatchProcesses(event, context, eventCallback) {
   const toDispatch =
     event.processes || (JOBS_SAME_PROCESS ? syncProcessMap : processMap);
-  await Promise.all(
+
+  const allResults = await Promise.all(
     Object.keys(toDispatch).map(p => {
+      if (process.env.JOB_PROCESS_MAP) {
+        const envProcessMap = String(process.env.JOB_PROCESS_MAP)
+          .replace(/ /g, "")
+          .split(",");
```

Comment:
could you pull this out to a constant like syncProcessMap? just so you don't have to re-declare it on every iteration

Reply:
good call, made the change!